unit-tests/live/options/test-uvc-power-stress-test.py

```diff
@@ -0,0 +1,70 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2025 Intel Corporation. All Rights Reserved.
+
+# test:device each(D400*)
```

Comment:
Do we want it on each PR or nightly only?

Reply:
It is not a long test, but the code does not change often. Will run it only nightly.